apps/nextjs-example/src/components/transactionFlows/MultiAgent.tsx

```diff
@@ -55,9 +66,9 @@ export function MultiAgent() {
       sender: account.address,
       secondarySignerAddresses: [secondarySigner.accountAddress],
       data: {
-        function: "0x1::coin::transfer",
-        typeArguments: [APTOS_COIN],
-        functionArguments: [account.address.toString(), 1], // 1 is in Octas
+        bytecode: TRANSFER_SCRIPT,
+        typeArguments: [parseTypeTag(APTOS_COIN)],
+        functionArguments: [AccountAddress.fromString(account.address), new U64(1)],
```

Comment:
`account.address` is now an `AccountAddress`
```suggestion
        typeArguments: [parseTypeTag(APTOS_COIN)],
        functionArguments: [account.address, new U64(1)],
```

Reply:
fixed